hokusai/production.yml

```diff
@@ -103,3 +103,75 @@ spec:
                     operator: In
                     values:
                     - background
+
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: opstools-cleanup-pods-by-name
+spec:
+  schedule: "0 * * * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      backoffLimit: 1
+      template:
+        metadata:
+          annotations:
+            "cluster-autoscaler.kubernetes.io/safe-to-evict": "false"
+        spec:
+          containers:
+            - name: opstools-cleanup-pods-by-name
+              image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/opstools:production
+              command: ["src/kubernetes_cleanup_pods_by_name/cleanup.py", "hokusai-run", "6", "--force"]
+              imagePullPolicy: Always
+              envFrom:
+                - configMapRef:
+                    name: opstools-environment
+          serviceAccountName: opstools
+          restartPolicy: Never
+          affinity:
+            nodeAffinity:
+              requiredDuringSchedulingIgnoredDuringExecution:
+                nodeSelectorTerms:
+                - matchExpressions:
+                  - key: tier
+                    operator: In
+                    values:
+                    - background
+
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: opstools-cleanup-pods-by-name
+spec:
+  schedule: "0 * * * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      backoffLimit: 1
+      template:
+        metadata:
+          annotations:
+            "cluster-autoscaler.kubernetes.io/safe-to-evict": "false"
+        spec:
+          containers:
+            - name: opstools-cleanup-pods-by-name
+              image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/opstools:production
+              command: ["src/kubernetes_cleanup_pods_by_name/cleanup.py", "jenkins", "168", --namespace data-application, "--force"]
```

Comment:
Do we still run anything in the `data-application` namespace? I suspect not, since migrating data pipelines to Airflow. In that case, maybe this whole cron can be eliminated.

Reply:
The namespace has now been deleted!

Reply:
Removed jenkins crons in https://github.com/artsy/opstools/pull/29/commits/806cb5a1c52f755e96acb83a97fabb70c6b24685